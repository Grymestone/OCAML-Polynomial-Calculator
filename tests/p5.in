2x^2 + 3x^7 - 3x^1
