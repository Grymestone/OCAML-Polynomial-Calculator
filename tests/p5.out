(((2*(x)^2)+(3*(x)^7))-(3*(x)^1))
2x^2 + 3x^7 + -3x^1